### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by VishnuVNairCodes**

### CODE REVIEW

File: components/typography.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/* || Headings || */
+.h1,
```
Comment (Parul1999) [inline components/typography.css:2]: What is the use of giving styling on classname as well as the element?

File: components/typography.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        <ul class="nav-links">
+          <li class="list">
+            <a href="/docs/docs.html" class="link nav-link">Docs</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="content">
+        <aside class="aside">
+          <ul>
+            <!-- <li class="list aside-heading">Components</li> -->
```
Comment (anandsr21harshit) [inline components/typography.html:34]: Unwanted comment at L-34

**Review by anandsr21harshit — APPROVED** [review]
You can remove unwanted comment. I have checked in codesandbox, looks good to merge.

File: components/typography.css (hunk 1 above)
Comment (anandsr21harshit) [reply components/typography.css:2]: I think he will use that class in while using text utilities.

**Review by KamiyaGaikwad — APPROVED** [review]
Good to merge

File: components/typography.css (hunk 1 above)
Comment (VishnuVNairCodes) [reply components/typography.css:2]: Yeah! I was thinking of giving provision to use these styles in p tag, div tag etc

File: components/typography.html (hunk 2 above)
Comment (VishnuVNairCodes) [reply components/typography.html:34]: Will take care of it in future👍

**Pull request merged**